ibis/tests/sql/conftest.py

```diff
@@ -11,15 +11,19 @@
 from ibis.tests.expr.mocks import MockBackend
 
 
-def sql_golden_check(funcname: str, path: Path, expr: ir.Expr | str) -> None:
```

Comment:
What do you think about using the `pytest-snapshot `package to write these tests? 
Thanks to this, we will be able to update the tests with one command.
```
pytest --snapshot-update.
```
We also won't have to have a separate file for each assertion, but we will have more assertions in one snapshot file.

Reply:
I put up #4836 to do this for our existing sqlgolden checks. I'll merge that PR when it's green.

Reply:
#4836 is merged!